app/api/id/[form]/submission/confirm/route.ts

```diff
@@ -130,7 +131,35 @@ async function confirm(
     }),
   });
 
-  await dynamoDBDocumentClient.send(request);
+  await sendWithExponentialBackoff(request);
+}
+
+async function sendWithExponentialBackoff(
```

Comment:
We might make this function more general by passing in a function instead of a specific request object, so we can use it with any AWS service that might give retryable errors. But, let's not overengineer it. Keep it simple until we really need more.